### ISSUE TITLE
Ketysian's Landing (DE Chapter 16 Easy PQ) Influence doesn't contribute to Chapter 16 INF progress

### ISSUE THREAD

**Issue by gefGitAcc1** [WarEmu/WarBugs#20823]
**Expected behavior and actual behavior:**
When killing mobs in the PQ there is influence score popuing on character. But these influence points don't contibute to Chapter progress

**Steps to reproduce the problem:**

1. Go to Ketysian's Landing (DE Chapter 16 Easy PQ) with no influence for DE Chapter 16
2. Kill mobs
3. Check DE Chapter 16 progress

**Testing Screenshots/Videos/Evidences (always needed):**

![Kidzadza_003](https://user-images.githubusercontent.com/1407580/228901304-b7b1beb3-780d-43ec-8ca1-89f15129edc0.jpg)

**Comment by lilimx** [comment]
Hey,

the influence bar for chapter  in your tome is empty aswell?

**Comment by gefGitAcc1** [comment]
Yes, I've checked it also. Indeed, PQ belongs to Chapter 15 Dark Elf, but progress bar is shown for Chapter 16 Dark Elf.

Progress is tracked in tome correctly to Chapter 15 DE. So issue is only in presenting correct Chapter on UI
